lib/Debug.php

```diff
@@ -62,9 +62,9 @@ class Debug {
 	public static function isEnabled() {
 		static $firstCall = true; // Initialized on first call
 
-		if($firstCall && file_exists(PATH_ROOT . 'DEBUG')) {
+		if($firstCall && file_exists(__DIR__ . '/../DEBUG')) {
```

Comment:
Why can't we use PATH_ROOT here and on other places?

Reply:
I guess this is mostly a matter of taste. I find the `__DIR__` and `__FILE__` more intuitive because they are file relative. Whereas the `PATH_ROOT` is unknown and must be looked up by new code reviewers.
In any case `PATH_ROOT` and `__DIR__ . '/../'` points to the same directory.

Reply:
Also when we migrate to composer's autoloader later (I hope!) we can't use `PATH_ROOT` in `composer.json` to define where our classfiles are located.

Reply:
> I guess this is mostly a matter of taste. I find the `__DIR__` and `__FILE__` more intuitive because they are file relative. Whereas the `PATH_ROOT` is unknown and must be looked up by new code reviewers. In any case `PATH_ROOT` and `__DIR__ . '/../'` points to the same directory.

Matter of taste is not a good reason for change. `__DIR__ . '/../'` does not tell you anything about the path, except its location. `PATH_ROOT` (as well as `PATH_LIB`, `PATH_CACHE`, etc.) were introduced to improve readability of the codebase by adding semantic information. Please also note that it is much easier to find all references of `PATH_ROOT` than it is to locate the right place in all variations of `__DIR__ . '/../...'`.

That said, this change is not needed for the autoloader and should be removed from this particular PR.

> Also when we migrate to composer's autoloader later (I hope!) we can't use `PATH_ROOT` in `composer.json` to define where our classfiles are located.

This is fine. Once composer's autoloader is implemented, most of the paths will vanish from the codebase anyway.